pennylane/transforms/classical_jacobian.py

```diff
@@ -152,21 +150,7 @@ def classical_preprocessing(*args, **kwargs):
 
     if qnode.interface == "autograd":
 
-        def _jacobian(*args, **kwargs):
-            if argnum is None:
-                jac = qml.jacobian(classical_preprocessing)(*args, **kwargs)
-            elif np.isscalar(argnum):
-                jac = qml.jacobian(classical_preprocessing, argnum=argnum)(*args, **kwargs)
-            else:
-                jac = tuple(
-                    (
-                        qml.jacobian(classical_preprocessing, argnum=i)(*args, **kwargs)
-                        for i in argnum
-                    )
-                )
-            return jac
-
-        return _jacobian
+        return qml.jacobian(classical_preprocessing, argnum=argnum)

```

Comment:
so nice 😍

Reply:
Right? This one made me really happy :D :)